0005-ordered-quantification.md

```diff
@@ -575,6 +591,24 @@ arbitrary Dafny types that provide a way to obtain an enumerator for their conte
 the necessary interface. This is also likely the best way to provide a better user experience
 for looping or quantifying over the values produced by an `iterator` type.
 
+## Range expressions
+
+This would interpret the existing `i..j` syntax, 
+currently only usable in sequence or array indexing expressions such as `s[i..j]`,
```

Comment:
Do you envision it would be possible to say something like this:
```
var x := i..j;
s[x]
```
and obtain a slice of `s`?
I personally was thinking that `i..j` would be available only in `[i..j]` or after an àrrow `<- i..j`.
I do love this syntax, but what about reverse iteration?

Reply:
Yes I was thinking it would be possible, just so we're not introducing a new type of value or special-casing a lot. I also like the syntax but have no good suggestion for the reverse order, and that's one reason this idea is safely quarantined in Future Possibilities for now. :)